src/components/Banner/Banner.jsx

```diff
@@ -0,0 +1,75 @@
+import {useMemo, useState} from "react";
+import CloseIcon from '@mui/icons-material/Close';
+import CampaignIcon from '@mui/icons-material/Campaign';
+import './Banner.css';
+
+export default function Banner(props) {
+    const [show, setShow] = useState(true);
+    const bannerPlace = useMemo(() => {
+        return props.bannerPlace === 'bottom' ? {
+            bottom: 0,
+            position: 'fixed',
+            width: '100%'
+        } : {}
+    }, [props.bannerPlace])
+    if (window.localStorage.getItem('seen_banner')) {
+        return <></>
+    }
+    return (show ?
+        <div style={{...props.bannerStyle, ...bannerPlace}}
+             className="bg-indigo-600">
+            <div className="max-w-7xl mx-auto py-5 px-5 sm:px-6 lg:px-8">
+                <div className="flex items-center justify-between flex-wrap">
+                    <div className="w-0 flex-1 flex items-center">
+            <span className="flex p-2 rounded-lg bg-indigo-800">
+              <CampaignIcon fontSize={"large"} className="h-8 w-8 text-white" aria-hidden="true"/>
+            </span>
+                        <p className="ml-3 font-medium text-lg text-white truncate">
+                            <span className="md:hidden">{props.bannerDescription}</span>
+                            <span
+                                className="hidden md:inline"><span
+                                className="font-bold">{props.bannerTitle}</span> {props.bannerDescription}</span>
+                        </p>
+                    </div>
+                    <div className="order-3 mt-2 flex-shrink-0 w-full sm:order-2 sm:mt-0 sm:w-auto">
+                        <a
+                            target="_blank"
+                            rel="noreferrer"
+                            onClick={props.bannerButtonOnClick}
+                            href={props.bannerButtonUrl}
+                            style={props.bannerButtonStyle}
+                            className={`flex items-center justify-center py-2 px-3 border border-transparent banner-button rounded-md text-md font-medium text-indigo-600 bg-white hover:bg-indigo-50 ${props.bannerButtonClass}`}
+                        >
+                            {props.bannerButtonText}
+                        </a>
+                    </div>
+                    <div className="order-2 flex-shrink-0 sm:order-3 sm:ml-3">
+                        <button
```

Comment:
Codacy has a fix for the issue: [Delete `············`](https://app.codacy.com/gh/mohammadnedaei/Nearby-Locations-React/pullRequest?prid=9855911)
```suggestion
            <button
```